### PR TITLE
[p6spy] Add documentation for the 'zipkinServiceName' int url feature.

### PR DESCRIPTION
**Pull request by hypnoce**

### CODE REVIEW

**Comment by hypnoce** [comment]
I'm not a native english speaker. Let me know if you find anything wrong with what's written.

File: instrumentation/p6spy/README.md [hunk 1]
```diff
@@ -33,4 +33,13 @@ includeParameterValues=true
 excludebinary=true
 ```
 
+`spy.properties` applies globally to any instrumented jdbc connection. If you need more flexibility to set the remoteServiceName you can add
```
Comment (adriancole) [inline instrumentation/p6spy/README.md]: `spy.properties` applies globally to any instrumented jdbc connection. To override this, add the `zipkinServiceName` property to your connection string.

**Comment by hypnoce** [comment]
Sorry many meetings today :(

**Comment by adriancole** [comment]
now worries. thanks a lot!

**Pull request merged**